.github/workflows/pr.yml

```diff
@@ -43,4 +43,5 @@ jobs:
         repo-token: "${{ secrets.GITHUB_TOKEN }}"
         title: 'File annotations for theory linter'
         input: './annotations.json'
+        disable: sorry
```

Comment:
The line has to go one higher into the previous step. I.e.
```
- uses: seL4/ci-actions/thylint@master
  with:
    disable: sorry
```

Reply:
Oh right, of course